### PR TITLE
Point rust-toolbox deps to enclone_ranger

### PR DESCRIPTION
**Pull request by macklin-10x**
Replaces the rust-toolbox dependency with the consolidated crates in enclone_ranger.

Pulls the `supp_regions.fa` files into this repo and includes them only in the single executable that uses them.

### CODE REVIEW

File: enclone_exec/Cargo.toml [hunk 1]
```diff
@@ -25,37 +25,37 @@ path = "src/bin/enclone.rs"
...
 enclone_testlist = { path = "../enclone_testlist" }
 enclone_visual = { path = "../enclone_visual", optional=true }
 flate2 = "1"
 home = "0.5"
-io_utils = { version = "0.3", git = "https://github.com/10XGenomics/rust-toolbox.git", branch = "master" }
+io_utils = { git = "https://github.com/10XGenomics/enclone_ranger", branch = "main" }
 itertools = "0.10"
-nix = "0.23"
-pretty_trace = { version = "0.5", git = "https://github.com/10XGenomics/rust-toolbox.git", branch = "master", features = ["pprof"]}
+nix = "0.27"
```
Comment (macklin-10x) [inline enclone_exec/Cargo.toml:36]: This was the only dependency version that needed to be tweaked.

File: enclone_tools/src/bin/annotate_seq.rs [hunk 2]
```diff
@@ -107,7 +114,17 @@ fn main() {
         let ext_ref = String::new();
         make_vdj_ref_data_core(&mut refdata, &refx, &ext_ref, true, true, None);
     } else {
-        make_vdj_ref_data(&mut refdata, imgt, &species, false, is_tcr, is_bcr);
+        println!("doin' it");
```
Comment (nmmsv10x) [inline enclone_tools/src/bin/annotate_seq.rs:117]: debugging comment? :)

Comment (macklin-10x) [reply enclone_tools/src/bin/annotate_seq.rs:117]: Lol thanks, forgot I added that!

Comment (macklin-10x) [reply enclone_tools/src/bin/annotate_seq.rs:117]:
```suggestion
```

**Pull request merged**